public/assets/js/scripts.js

```diff
@@ -322,101 +333,117 @@ $(document).ready(function () {
+      output.push(categories.hybrid);
     }
-    
-    if ($(window).width() < MOBILE_WIDTH) {
-      for (let i = 0; i < output.length; i++) {    
+
+    for (let i = 0; i < output.length; i++) {
+      if ($(window).width() < MOBILE_WIDTH) {
         recommendationBody += `<div class="card recommendation-card">
-                                <div class="card-header" id="heading${i}">
-                                  <a data-toggle="collapse" href="#collapse${i}" aria-expanded="false" aria-controls="collapse${i}">
-                                    <h6 class="mb-0">
-                                      ${output[i].title}
-                                      <span class="recommendation-card-icon">&nbsp; ⌄ </span>
-                                    </h6>
-                                  </a>
-                                </div>
-                                <div id="collapse${i}" class="collapse" aria-labelledby="heading${i}" data-parent="#accordion">
-                                  <p class="heading">Requirements</p>
-                                  <div class="card-body">${output[i].message}</div>
-                                  <p class="target">Target:</p>
-                                  <h4 class="price">NGN ${formatToCurrency(avgPropertyCost)}</h4>
-                                </div>
-                              </div>`;
-      }
-      recommendation = `<div id="accordion">
-                          ${recommendationBody}
-                        </div>`;
-    } else {
-      for (let i = 0; i < output.length; i++) {
+                                  <div class="card-header recommendation-card-header" id="heading${i}">
+                                    <a data-toggle="collapse" href="#collapse${i}" aria-expanded="false" aria-controls="collapse${i}">
+                                      <h6 class="mb-0">
+                                        ${output[i].title}
+                                        <img src="./assets/img/icons/question-mark.svg" alt="payment">
+                                        <span class="recommendation-card-icon">&nbsp; ⌄</span>
+                                      </h6>
+                                    </a>
+                                  </div>
+                                  <div id="collapse${i}" class="collapse" aria-labelledby="heading${i}" data-parent="#accordion">
+                                    <p class="heading">Requirements</p>
+                                    <div class="card-body">${output[i].message}</div>
+                                    <p class="target">Target:</p>
+                                    <h4 class="price">NGN ${formatToCurrency(avgPropertyCost)}</h4>
+                                  </div>
+                                </div>`;
+
+        recommendation = `<div id="accordion">
+                            ${recommendationBody}
+                          </div>`;
+      } else {
         if (i == 0) {
```

Comment:
There are lots of repetitions between the code block in the if statement and the else statement. Can you use the if (i === 0) only where the differences are between the two blocks? I think the only difference is adding the active and show class to the first accordion which could be resolved with a single liner like this:
```js
// line 411
recommendationBody += `<div class="tab-pane fade ${i === 0 ? 'active show' : ''} ... >`

```